compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1052,6 +1052,10 @@ class Allocator {
                                 void *Block, const uptr UserPtr,
                                 const uptr SizeOrUnusedBytes,
                                 const FillContentsMode FillContents) {
+    // Compute the default pointer before adding the header tag
+    const uptr DefaultAlignedPtr =
+        reinterpret_cast<uptr>(Block) + Chunk::getHeaderSize();
+
```

Comment:
Nice catch!

I think we also need to fix the `DefaultAlignedPtr` in initChunkWithMemoryTagging, right?